README.md

```diff
@@ -18,6 +18,7 @@ A GitHub action to turn a GitHub project into a self-hosted Helm chart repo, usi
 - `version`: The chart-releaser version to use (default: v1.7.0)
 - `config`: Optional config file for chart-releaser. For more information on the config file, see the [documentation](https://github.com/helm/chart-releaser#config-file)
 - `charts_dir`: The charts directory
+- `repository`: The repository to release (default: the repository the action is running from). If used - make sure to set the appropriate CR_TOKEN for the targeted repository.
```

Comment:
maybe make it clear that need a  `org/repo` and in the `default`. add

```
 (default: $GITHUB_REPOSITORY - the repository the action is running from)
```

Reply:
Sure, fixed